.github/ubiquibot-config.yml

```diff
@@ -3,3 +3,8 @@ price-multiplier: 1.5
 default-labels:
   - "Time: <1 Hour"
   - "Priority: 1 (Normal)"
+  first-time-contributor-greeting:
+  enabled: true
+  header:  "Thank you for contributing to UbiquiBot! Please be sure to set your wallet address before completing your first bounty so that the automatic payout upon task completion will work for you."
+  helpMenu: true
+  footer:  "###### Also please star this repository and [@ubiquity/devpool-directory](https://github.com/ubiquity/devpool-directory/) to show your support. It helps a lot!"
```

Comment:
yml doesn't appear to be correct.

Reply:
I was also wondering what happened to it

Reply:
In the original description we had to add it to ubiquibot-config.yml but where should I add it now

Reply:
Most configs have been moved to org config but you are adding it to the correct file.  Just the yaml is invalid and you need to fix that.